src/main/java/com/google/sps/data/PersistentUserRepository.java

```diff
@@ -0,0 +1,163 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Query.CompositeFilter;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import java.lang.IllegalArgumentException;
```

Comment:
nit: this can be moved lower in the page to keep this ascii sorted.

Reply:
Which import, line 12? or all the imports?

Reply:
Oh u mean lower it bc its not in ascii order. Gotcha

Reply:
Done